homeassistant/components/rainbird/services.yaml

```diff
@@ -19,3 +19,16 @@ start_irrigation:
           min: 1
           max: 1440
           unit_of_measurement: "minutes"
+set_rain_delay:
+  name: Set rain delay
+  description: Set how long automatic irrigation is turened off
+  fields:
+    duration:
+      name: Duration
+      description: Duration for this system to be turned off
```

Comment:
```suggestion
      description: Duration for this system to be turned off.
```

Reply:
The dot  at the end is already added in the last commit.